### ISSUE TITLE
Bad coloring of comment after multi-line initializer list

### ISSUE THREAD

**Issue by kylrth** [jeff-hykin/cpp-textmate-grammar#305]
# Issue Type: Bug

## To reproduce

Type this in a C++ header file (e.g. `sample.hpp`) in VS Code:

```cpp
class ThisTest {
	ThisTest()
	    : someMember(true),  // here is a comment that is formatted incorrectly
	      anotherOne(false),
	      quiteALotOfMembers(false),  // here's another one
	      soManyThatTheConstructorIsLong(true),
	      iMeanReallyLong(false) {
	}

	bool someMember;
	bool anotherOne;
	bool quiteALotOfMembers;
	bool soManyThatTheConstructorIsLong;
	bool iMeanReallyLong;
};
```

## Expected
The line comments should be correctly highlighted.

## Actual
The line comments are formatted with the same color as the parentheses.

## Screenshot

![bad highlighting](https://user-images.githubusercontent.com/5044802/61137944-208d0400-a48c-11e9-85e2-81e17384cddc.png)

In GitHub, the code is correctly highlighted.

Setting `C_Cpp.enhancedColorization` to enabled or disabled changes nothing.

## Configuration

C/C++ extension version: 0.24.0
VS Code version: 1.37.0-insider (user setup)
Commit: 46cc0a1670b7047e610aaef1bf84f1190802e9ec
Date: 2019-07-11T05:39:42.001Z
Node.js: 10.11.0
OS: Windows_NT x64 6.3.9600 (Windows 8)

**Comment by matter123** [comment]
What version are you using?

**Comment by kylrth** [comment]
How can I tell? Sorry.

**Comment by matter123** [comment]
If you installed the extension it should show it in the extension manager. If your using vscode included version that is v1.12.11.

**Comment by kylrth** [comment]
Haha OK, I get it now. I just installed your extension and the problem persists. Version: 1.12.19.

**Comment by jeff-hykin** [comment]
Okay should be fixed with `v1.12.20`, let me know if it persists
